src/mapping.hpp

```diff
@@ -1,198 +1,12 @@
-)";
\ No newline at end of file
+extern const std::string keystonesJson;
+extern const std::string secondaryRunesJson;
```

Comment:
see the extern definitions here. This is the way to write a header for a declared but not defined constant, where the constant is defined elsewhere (an external-to-this-file compilation unit). `extern` is the developer promising that the linker will be able to find the necessary definition. (Of course, if the linker can't find it, there would be a linker error, so there's no runtime-only errors as a result of doing this pattern.)